### ISSUE TITLE
[error] [net_module.cpp:37] Net backend not found: onnxruntime

### ISSUE THREAD

**Issue by Adenialzz** [open-mmlab/mmdeploy#344]
I build a docker image with [dockerfile-gpu](https://github.com/open-mmlab/mmdeploy/blob/master/docker/GPU/Dockerfile), and follow the instruction in [get started](https://mmdeploy.readthedocs.io/en/latest/get_started.html). All steps are fine, until the last step, when running the executable `object_detection`, I get an error like this:
```
[2022-04-14 01:34:43.313] [mmdeploy] [info] [cuda_device.cpp:84] Default CUDA allocator initialized
[2022-04-14 01:34:43.313] [mmdeploy] [error] [net_module.cpp:37] Net backend not found: onnxruntime
[2022-04-14 01:34:43.313] [mmdeploy] [error] [detector.cpp:58] exception caught: entry not found (8) @ /root/workspace/mmdeploy/csrc/net/net_module.cpp:38
failed to create detector, code: 6
```

I have already set environment variable `PATH` and `LD_LIBRARY_PATH` about onnxruntime lib:

```shell
root@5ee45f96fd7b:~/workspace/mmdeploy/build/install/example/build# echo $PATH
/root/workspace/onnxruntime-linux-x64-1.8.1/lib/:/opt/conda/bin:/opt/tensorrt/bin:/usr/local/mpi/bin:/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/ucx/bin
root@5ee45f96fd7b:~/workspace/mmdeploy/build/install/example/build# echo $LD_LIBRARY_PATH
/root/workspace/onnxruntime-linux-x64-1.8.1/lib/:/root/workspace/mmdeploy/build/lib:/usr/local/cuda/compat/lib:/usr/local/nvidia/lib:/usr/local/nvidia/lib64
root@5ee45f96fd7b:~/workspace/mmdeploy/build/install/example/build# ls /root/workspace/onnxruntime-linux-x64-1.8.1/lib/
libonnxruntime.so  libonnxruntime.so.1.8.1
```

So how can I fix this problem? thanks in advance.

**Comment by RunningLeon** [comment]
@Adenialzz Hi,could you try add `-DONNXRUNTIME_DIR=${ONNXRUNTIME_DIR}` in blew here:
https://github.com/open-mmlab/mmdeploy/blob/6e7e219b0b232d45aff58a6c8a23ef4fc047ec12/docker/GPU/Dockerfile#L84

**Comment by RunningLeon** [comment]
> > @Adenialzz Hi,could you try add `-DONNXRUNTIME_DIR=${ONNXRUNTIME_DIR}` in blew here:
> > https://github.com/open-mmlab/mmdeploy/blob/6e7e219b0b232d45aff58a6c8a23ef4fc047ec12/docker/GPU/Dockerfile#L84
> 
> Thank you for reply. I tried your advice, but it still not work. Maybe I should also change `-DMMDEPLOY_TARGET_BACKENDS="trt"` to `-DMMDEPLOY_TARGET_BACKENDS="ort"` ?

@Adenialzz 
Yes you are right please update to  `DMMDEPLOY_TARGET_BACKENDS="ort;trt" `.

**Comment by Adenialzz** [comment]
> > > @Adenialzz Hi,could you try add `-DONNXRUNTIME_DIR=${ONNXRUNTIME_DIR}` in blew here:
> > > https://github.com/open-mmlab/mmdeploy/blob/6e7e219b0b232d45aff58a6c8a23ef4fc047ec12/docker/GPU/Dockerfile#L84
> > 
> > 
> > Thank you for reply. I tried your advice, but it still not work. Maybe I should also change `-DMMDEPLOY_TARGET_BACKENDS="trt"` to `-DMMDEPLOY_TARGET_BACKENDS="ort"` ?
> 
> @Adenialzz Yes you are right please update to `DMMDEPLOY_TARGET_BACKENDS="ort;trt" `.

OK, it works now. By the way, what is the reason for not putting 'ort' support in dockerfile?

**Comment by RunningLeon** [comment]
@Adenialzz Hi, it could be a bug here. @AllentDan, is it intended to not compile `ort` in GPU dockerfile?

**Comment by Adenialzz** [comment]
> @Adenialzz Hi, it could be a bug here. @AllentDan, is it intended to not compile `ort` in GPU dockerfile?

Hi, for these changes, I have done testing and submitted a PR to the gpu dockerfile. Still awaiting review. See #366 .

**Comment by RunningLeon** [comment]
@Adenialzz Perfect.

**Comment by RunningLeon** [comment]
Fixed in #366
